agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java

```diff
@@ -317,6 +317,8 @@ public static class PreviewConfiguration {
     public boolean disablePropagation;
     public boolean captureHttpServer4xxAsError = true;
 
+    public static InjectJavaScriptSnippet injectJavaScriptSnippet = new InjectJavaScriptSnippet();
```

Comment:
```suggestion
    public InjectJavaScriptSnippet injectJavaScriptSnippet = new InjectJavaScriptSnippet();
```

Reply:
injectJavaScriptSnippet would be called at runtimeConfigurator, if remove static, then would encounter error message "Non-static field 'injectJavaScriptSnippet' cannot be referenced from a static context"